bindings/python/docs/source/comparison.rst

```diff
@@ -0,0 +1,160 @@
+Quick Start
+===========
+
+This tutorial is intended as a comparison between using just PyMongo, versus
+with **PyMongoArrow**. The reader is assumed to be familiar with basic
+`PyMongo <https://pymongo.readthedocs.io/en/stable/tutorial.html>`_ and
+`MongoDB <https://docs.mongodb.com>`_ concepts.
+
+
+Reading Data
+^^^^^^^^^^^^
+The most basic way to read data using PyMongo is::
+
+.. code-block:: python
+
+    coll = db.benchmark
+    f = list(coll.find({}, projection={"_id": 0}))
+    table = pyarrow.Table.from_pylist(f)
+
+This works, but we have to exclude the "_id" field because otherwise we get this error::
+
+    pyarrow.lib.ArrowInvalid: Could not convert ObjectId('642f2f4720d92a85355671b3') with type ObjectId: did not recognize Python value type when inferring an Arrow data type
+
+The workaround gets ugly (especially if you're using more than ObjectIds):
+
+.. code-block:: pycon
```

Comment:
Why pycon instead of just the `::` shorthand?

Reply:
For syntax hightighting